horace_core/Tobyfit/detector_array.m

```diff
@@ -28,11 +28,12 @@
 % Because detpar only contains minimal information, either hardwire in
 % the detector type here or use the info now available in the detector
 % arrays
-detpar = wtmp.detpar();   % just get a pointer
+% detpar = wtmp.detpar;   % just get a pointer
 det = wtmp.experiment_info.detector_arrays;
 if isempty(det) || det.n_runs == 0
     % no detector info was available when the sqw was populated, so
     % continue with the old detector initialisation from detpar
+    error('trying to make detectors from a detpar but it doesnt exist any more');
```

Comment:
we have argeed to throwh errors with signature `error('HORACE:level2_location:invalid_argument',message)`

Reply:
fixed